.env.example

```diff
@@ -0,0 +1,2 @@
+PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```

Comment:
this is a dummy private key generated by [anvil](https://book.getfoundry.sh/reference/anvil/)